src/main/java/com/apac/test/apactakehometest/controller/CrudAPIController.java

```diff
@@ -0,0 +1,68 @@
+package com.apac.test.apactakehometest.controller;
+
+import com.apac.test.apactakehometest.model.TaxiTripsModel;
+import com.apac.test.apactakehometest.repository.TaxiTripsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/taxitrips")
+public class CrudAPIController {
+
+    @Autowired
+    private TaxiTripsService mTaxiTripsService;
+
+    @GetMapping(value = "all")
```

Comment:
we dont need to add `all` or `delete`
in REST we have a convention to use HTTP verb 
eg: 
GET `/` to findAll
GET `/{id}` getOne
POST `/` to create an object
PUT `/` to update an object 
DELETE `/{id}` to detele an object

please go through the article here : https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9

Reply:
Done

Reply:
please push your changes :)